packages/cli/src/oclif/commands/cache/clear.js

```diff
@@ -0,0 +1,111 @@
+const BaseCommand = require('../../ZapierBaseCommand');
+const { buildFlags } = require('../../buildFlags');
+const { listVersions, getWritableApp, callAPI } = require('../../../utils/api');
+const { cyan } = require('colors/safe');
+
+class ClearCacheCommand extends BaseCommand {
+  async perform() {
+    this.startSpinner('Fetching versions...');
+    const { versions } = await listVersions();
+    this.stopSpinner();
+
+    const { majorVersion } = this.args;
+
+    let selectedMajorVersion =
+      majorVersion === '' ? null : Number(majorVersion);
```

Comment:
I got this error when I ran `zapier cache:clear`:

```
$ zapier-dev cache:clear
✔ Fetching versions...
 ›   Error: This app does not have any versions on major version 'NaN'. Valid versions are: 2, 1
 ›
 ›   re-run this command with `--debug` for more info
```

When executing `zapier cache:clear` without args, `majorVersion` is `undefined`. So after this line, `selectedMajorVersion` will be `Number(undefined)`, which is `NaN`.

To fix, we should:
1. turn any falsey value (`undefined`, `null`, `''`) to `null`.
2. check any string that can't be parsed into a number (such string makes `Number(s)` a `NaN`)

```suggestion
    let selectedMajorVersion = majorVersion ? Number(majorVersion) : null;
    if (Number.isNaN(selectedMajorVersion)) {
        throw new Error(`Invalid major version '${majorVersion}'. Must be a number.`);
    }
```